members/state_il_gov_bruce_rauner.yaml

```diff
@@ -6,115 +6,83 @@ contact_form:
-            - "1518": "Film Tax Credits"
-            - "1519": "Fracking"
-            - "1520": "Funding for Amtrak Train Services"
-            - "1521": "Funding for Domestic Violence Programs"
-            - "1523": "Funding for Energy Assistance Programs"
-            - "1524": "Funding for Nursing Homes"
-            - "1529": "Gambling Expansion"
-            - "1530": "Gas Prices"
-            - "1531": "Gun Control"
-            - "1539": "Healthcare Reform"
-            - "1540": "High Speed Rail"
-            - "1541": "Higher Education Funding"
-            - "1542": "Homeless Services"
-            - "1543": "IDOT South Suburban/Peotone Airport"
-            - "1545": "Immigration Reform"
-            - "1546": "Independent Maps"
-            - "1547": "Infrastructure Funding"
-            - "1551": "Medicaid Reductions"
-            - "1552": "Medical Marijuana"
-            - "1553": "Mental Health Funding"
-            - "1554": "Minimum Wage Increase"
-            - "1556": "Pension Reform"
-            - "1557": "Prison Reform"
-            - "1558": "Property Tax Freeze"
-            - "1560": "Public Education Funding "
-            - "1561": "Public Transit Funding"
-            - "1562": "Renewable Energy "
-            - "1563": "Right to Work"
-            - "1571": "Social Services Budget Cuts"
-            - "1572": "State Income Tax"
-            - "1573": "Supportive Housing"
-            - "1574": "Tax Reform"
-            - "1575": "Term Limits"
-            - "1578": "the Governor’s Proposed Budget Reductions"
-            - "1576": "The Governor's Job Performance"
-            - "1581": "Updating the Illinois Communications Law"
-            - "1582": "Vehicle Emissions Testing"
-            - "1583": "Welfare Reform"
-            - "1584": "Workers Compensation Reform"
-            - "1585": "Youth Program Funding"
-            - "36": "(Other - Subject not listed)"
+            - "1": "Agriculture"
+            - "2": "Animal Welfare"
+            - "3": "Arts & Humanities"
+            - "4": "Budget"
+            - "5": "Business & Economy"
+            - "6": "Children's Health & Welfare"
+            - "7": "Disability Services"
+            - "8": "Education"
+            - "9": "Energy & Environment"
+            - "10": "Government Reform & Oversight"
+            - "11": "Health & Healthcare"
+            - "12": "Human Rights"
+            - "13": "Immigration"
+            - "14": "Labor & Employment"
+            - "15": "Public Safety"
+            - "16": "Social Services"
+            - "17": "Senior Services"
+            - "18": "Taxes"
+            - "19": "Transportation & Infrastructure"
+            - "20": "Veterans & Military Affairs"
+            - "21": "Voting"
+            - "22": "Other Issues"
     - click_on:
         - value: Submit
-          selector: "#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_SOIFeedbackSubmit"
+          selector: input[type='submit'][value='Submit']
```

Comment:
they removed the ID for the submit button on the form?

Reply:
Well not really. The computer generated ID is there, but Anna showed me this way to select the submit button a few weeks ago. I just think this is a safer for when the form is updated with new selectors. At least we know that we submit button is working. Thoughts?

Reply:
in this case, if there is an ID on the button, we should use that. the more generic way , which you have, can be used if there is nothing else better. if these IDs are computer generated, the selectors for the other fields will fail, so changing this to what you have doesn't really get us anything. something better could be `submit#ctl00_m_g_3a01d56e_b68a_485b_a7d3_43e25d8506d3_SOIFeedbackSubmit`

Reply:
Understood. I'll change it and add the ID for the button